src/policies/InvalidationPolicyManager.ts

```diff
@@ -101,9 +101,19 @@ export default class InvalidationPolicyManager {
     if (!typePolicyForEvent) {
       return;
     }
-    Object.keys(typePolicyForEvent).forEach((typeName: string) => {
-      const typeMapEntities = entityTypeMap.readEntitiesByType(typeName) ?? {};
-      const policyAction = typePolicyForEvent[typeName];
+
+    const { __default: defaultPolicyAction, ...restTypePolicyTypeNames } = typePolicyForEvent;
+
+    if (defaultPolicyAction) {
+      defaultPolicyAction(mutedCacheOperations, {
+        storage: this.getPolicyActionStorage(`${typeName}__default`),
+        ...policyMeta,
+      });
```

Comment:
@danReynolds Not sure I follow exactly what this does function call accomplishes. `mutedCacheOperations` is an instance variable, is that correct?

Reply:
Yea so this part of the code is where we call the invalidation policy action handlers:

```
Employee: {
  onWrite: {
     EmployeeMessage: () => {// call the user provided function for employee messages on write of an employee }
  }
}
```

and so now we're adding support for a `__default` key so that you can trigger side-effects whenever the Employee gets written, without needing to say that it be for an `EmployeeMessage`, but rather just as a generic side effect. The `mutedCacheOperations` are the `readField`, `evict` and `modify` APIs that we provide to those policy action handlers